qbft/controller.go

```diff
@@ -58,7 +37,7 @@ func NewController(
 		Height:              -1, // as we bump the height when starting the first instance
 		Domain:              domain,
 		Share:               share,
-		StoredInstances:     InstanceContainer{},
+		StoredInstances:     NewInMemContainer(HistoricalInstanceCapacity),
```

Comment:
It should receive the instance via the func params not hardcoded

Reply:
just aligned the old code.. no change has made